demo/src/screens/componentScreens/HintsScreen.tsx

```diff
@@ -85,6 +89,17 @@ export default class HintsScreen extends Component<HintScreenProps, HintScreenSt
             // borderRadius={BorderRadiuses.br40}
             // edgeMargins={30}
             // onBackgroundPress={() => this.setState({showHint: !showHint})}
+            customContent={
+              showCustomContent ? (
+                <Text text70 white>
+                  Click
+                  <Text onPress={() => Alert.alert('custom content :)')} text70BO violet50>
```

Comment:
Not sure why this is not working on Android (does not look like it's [this](https://github.com/facebook/react-native/issues/22747)), consider changing the example to a button so this works properly.

Reply:
Removing the code you've mentioned (below) makes me **very** uncomfortable, adding the `collapsable={false}` usually comes in to fix bugs, the `zIndex` most likely fixes a bug as well, we can look it up (in the tickets or Slack) if you'd like.
And the comment **mentions** it fixes a bug in Android.
```
collapsable={false}
---
zIndex: 100,
// This is a hack to make hint render correctly on Android
borderWidth: 1,
borderColor: 'transparent'
```

Reply:
Right, but it looks like a little bit old code and Ethan said it might be solved by `react-native` since then. 
Anyway, I did check it in Android and in iOS, and we can also make a version for QA before releasing it.
What do you think?

Reply:
OK, let's create a version and test it.

Reply:
Tested by Yulia and it looks as expected :)

Reply:
Cool 👍